lib/tbot/internal/diagnostics/service.go

```diff
@@ -131,7 +139,30 @@ func (s *Service) Run(ctx context.Context) error {
 		}
 	}()
 
-	if err := srv.ListenAndServe(); !errors.Is(err, http.ErrServerClosed) {
+	// When the process exists, the socket files are left behind (to cover
```

Comment:
Did you mean "when the process **exits**" here?

Reply:
Yes, thanks! (Fixed on the teleport side also.)